torchtitan/protocols/model.py

```diff
@@ -70,3 +71,21 @@ def get_attention_masks(
         raise NotImplementedError(
             "This model does not support attention masking/Flex Attention."
         )
+
+    def _get_varlen_attention_masks(
```

Comment:
These 2 functions are not needed in model protocol, because they are only called by `get_attention_masks` internally, and a model not necessary need to implement  these 2 methods